src/Compilers/VisualBasic/VbcCore/vbc.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+THISDIR=$(dirname $0)
+
+chmod +x $THISDIR/CoreRun 2>/dev/null
+$THISDIR/CoreRun $THISDIR/vbc.exe $@
```

Comment:
In order to pass all command line arguments as they were passed in, you must use `"$@"` instead of `$@`.

This one has bit me many times in the past.